python/ql/test/experimental/query-tests/Security-new-dataflow/CWE-089/SqlInjection.expected

```diff
@@ -0,0 +1,3 @@
+edges
+nodes
+#select
```

Comment:
Ah, I expected the django sources to be merged by the time this got in, so that we would have results. But if flask is preferred, I will get sources right away :-)

Reply:
Preference for `flask` over `django` is certainly debatable, it's mainly just my personal opinion :wink: but I hope you agree? :pray: